Hazel/src/Hazel/Utility/Random.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+#include <random>
+
+namespace Hazel {
+
+	class Random
+	{
+	public:
+		/*
+		* Returns a Random Real between min and max
+		*/
+		template<typename T>
+		static T Real(T min, T max)
+		{
+			HZ_CORE_ASSERT(min < max, "min is >= max");
```

Comment:
min max can conflict with std::min and std::max. Why not renaming to `low` and `high` ? Like this instead of the assert you can do std::uniform_real_distribution<T> dist(std::min(low, high), std::max(low, high));

Reply:
they will not confict since they are parameters and there is no functions called min() and max() as I'm aware. sd::min and std::max are namespaced functions and are different from just min and max unless you are doing `using namespace std;`, please don't.